FILE packages/desktop/src-tauri/Cargo.lock
RANGE {"path":"packages/desktop/src-tauri/Cargo.lock","old_lines":[2539,2543],"new_lines":[2539,2543],"diff_block_lines":[5,11],"hunk_header":"@@ -2539,5 +2539,5 @@ dependencies = [","substantive_changed_lines":2}
@@ -2539,5 +2539,5 @@ dependencies = [
 [[package]]
 name = "paseo"
-version = "0.1.13"
+version = "0.1.14"
 dependencies = [
  "log",
